src/__internal__/form-field/form-field.component.tsx

```diff
@@ -190,7 +191,7 @@ const FormField = ({
             inline={inlineLabel}
             width={labelWidth}
             optional={isOptional}
-            useValidationIcon={useValidationIcon}
+            useValidationIcon={!validationRedesignOptIn && useValidationIcon}
```

Comment:
**question**: Again, was this deliberate that it affects other input components besides just `switch`. If you use the `validationOnLabel` prop with the validation redesign flag, the current behaviour is shown in this sandbox https://codesandbox.io/s/gallant-wave-qpr3yk?file=/src/App.tsx - which could well be wrong as it has the old-style tooltip validation despite the flag (but we'd need UX confirmation of that).

In your branch, the same code now shows "new style" validation - so arguably that's a fix/improvement. But it's no longer clear that `validationOnLabel` actually does anything, so I think there's a bit more to it perhaps. And even if this is a needed fix for new validation in general it probably shouldn't be done in a PR that's supposed to be just about updating `switch`.